content/metadata.yaml

```
@@ -1372,3 +1372,22 @@ authors:
         order: 6
         contributions:
           - Writing - Review & Editing
+          
+          - 
+    name: Ariel I. Mundo
+    affiliations:
+      - Department of Biomedical Engineering, University of Arkansas, Fayetteville, Arkansas, USA
+    email: aimundo@uark.edu
+    orcid: 0000-0002-6014-4538
+    coi:
+      string: "TBD"
```

```suggestion
      string: "None"
```
Do you have any (potential) conflicts of interest?  If not, then we'll enter "None" here.

---

I have no conflicts of interest